spec/specification/json_spec.rb

```diff
@@ -86,11 +86,13 @@ module Pod
             'name' => 'GreenBanana',
             'source_files' => 'GreenBanana',
             'dependencies' => { 'AFNetworking' => [] },
+            'configuration_pod_whitelist' => { 'AFNetworking' => [] },
```

Comment:
@segiddins question from the original draft: It seems the original draft PR always set `attributes_hash['configuration_pod_whitelist']` whether any configurations were present or not. Are we OK with that or are unnecessarily "polluting" the JSON result?

Reply:
nm I think I was wrong, we were not setting this unless we had some whitelisted configs. Updating.